test/powershell/Host/Startup.Tests.ps1

```diff
@@ -21,6 +21,7 @@ Describe "Validate start of console host" -Tag CI {
             'System.ComponentModel.dll'
             'System.ComponentModel.Primitives.dll'
             'System.ComponentModel.TypeConverter.dll'
+            'System.Configuration.ConfigurationManager.dll'
```

Comment:
It's not worth it to add this converter to the cache at the very beginning.
This dll is 959 kb, almost 1mb, and this converter will rarely to be used (only for compatibility purpose). Pulling in the dll at startup time is not right.

Reply:
Yes :-). Will remove at all.
 I wonder why did Windows PowerShell add this in cache?

Reply:
We could add new converter using TypeTable_Types_Ps1Xml.cs like `Microsoft.PowerShell.DeserializingTypeConverter` but I guess that typeof(CommaDelimitedStringCollection) reference in the converter will load the assembly at startup.

Reply:
@daxian-dbw @SteveL-MSFT Could you please make a conclusion should we add back the converter?

.Net Core (and .Net Framework) does not use it on CommaDelimitedStringCollection by default. I don't know why it was added in Windows PowerShell.
I think we could remove the code.

Reply:
Since PSCore6 didn't have this and I haven't seen any complaints about it along with your GitHub search it seems unlikely anyone missed that this was gone.  So it seem we should not add it back given the size.

Reply:
@SteveL-MSFT that's what we were waiting for 😄

Reply:
Great, then this PR can be closed 😄

Reply:
I removed the old code in the PR and updated the PR description.